### ISSUE TITLE
[BLUS30147] Rock Band 2 Deluxe - Continuous searching for random players online

### ISSUE THREAD

**Issue by HERO3LIFE** [RPCS3/rpcs3#11372]
## Quick summary
While searching for random players using the search function online, I am never able to connect to any player, even while I am online and connected using the Network tab in rpcs3. 

## Details
The settings are pretty straight forward, under the Network tab in the custom configuration of RB2 Deluxe, I have (Network Status: Connected, PSN Status: Connected, and DNS set to 8.8.8.8.) The IP/Host Switches is a newly created server called "hmxcertps3.hmxservices.com=54.165.123.154". This was really the only requirement besides finding an old PS3 username of someone who has played the game before and creating an account using one of those accounts. I created my account successfully on the RPCN tab under Configuration and can log in without issue. The name shows up in the save function in-game. But, as I navigate to search for players online, I constantly keep searching. I have troubleshooted with many other players who can successfully connect, and they will connect with each other after a few seconds, but I never find them, even when they have the same settings as me. I have tried re-installing RPCS3, Reinstalling RB2 itself, and also I made sure the firewall settings weren't blocking anything. 

* OS Windows 10
* CPU Ryzen 7 1700x
* GPU EVGA GeForce RTX 2070 XC
[RPCS3.log.gz](https://github.com/RPCS3/rpcs3/files/7866308/RPCS3.log.gz)

**Comment by digant73** [comment]
@HERO3LIFE probably still an issue, but please provide an update

**Comment by carlmylo** [comment]
I don't think this is valid at the moment. There are no active servers for this game anymore and there currently aren't any plans to revive them.
